.github/ISSUE_TEMPLATE/add-orbit-chain-request.yml

```diff
@@ -169,7 +169,7 @@ body:
   - type: markdown
     attributes:
       value: |
-        ## Parent chain token bridge contract addresses
+        ## Parent chain (L2) token bridge contract addresses
```

Comment:
Let's remove L2 and L3 text because we also support L1<>L2 and it's confusing for the teams.

The ticket is a bit outdated and this part:
```
Fix headers so when issue is created L2 and L3 addresses can be distinguished, currently they are listed one under another without headers
```

referred to parent and chain contracts having the same headers but you've already fixed it in the automation PR